spec/indico_batch_spec.rb

```diff
@@ -75,7 +76,7 @@
     ])
 
     data = ['Quis custodiet ipsos custodes', 'Clearly english, foo!']
-    response = Indico.batch_language(data, @config)
+    response = Indico.batch_language(data)
```

Comment:
The @config is an artifact from before the ruby library could grab the api from environment variables or a config file. This way should pass if they have either set up, not just the enviroment variable.